hypothesis-python/src/hypothesis/extra/django/_fields.py

```diff
@@ -51,6 +65,18 @@
     dm.NullBooleanField: st.one_of(st.none(), st.booleans()),
     dm.URLField: urls(),
     dm.UUIDField: st.uuids(),
+    df.BooleanField: st.booleans(),
+    df.BooleanField: st.sampled_from(["0", "1"]),
+    df.DateField: st.dates(),
+    df.DateTimeField: st.datetimes(timezones=get_tz_strat()),
```

Comment:
I'm making a separate function for the form and model time fields primarily because the form shouldn't care whether or not sqlite is in use. The `DateTimeField`s are unified though.

Reply:
wow, I really didn't notice that I had duplicated the `BooleanField`... I've resolved that too.